### ISSUE TITLE
Needs to implement accessibility standards

### ISSUE THREAD

**Issue by gilesdring** [odileeds/example-repo#2]
Should conform to to WCAG AA

**Comment by slowe** [comment]
This should be done

**Comment by gilesdring** [comment]
THIS IS NOW DONE
